app/templates/components/nypr-brick-item.hbs

```diff
@@ -0,0 +1,23 @@
+<li class="{{item.attributes.template}} item--trending"
```

Comment:
can these be pathless keynames, e.g. `template` instead of `item.attributes.template`?